package.json

```diff
@@ -41,23 +41,26 @@
   },
   "dependencies": {
     "async": "0.2.9",
+    "browserify": "^16.2.2",
     "commander": "2.0.0",
-    "connect": "2.30.2",
-    "download-github-repo": "0.1.3",
+    "connect": "^3.6.6",
+    "download-github-repo": "^0.1.4",
     "envy-json": "0.2.1",
     "escape-html": "1.0.3",
-    "fs-extra": "0.18.2",
+    "fs-extra": "1.x",
+    "marked": "^0.5.1",
     "mime": "1.3.6",
     "parseurl": "1.3.0",
     "pause": "0.1.0",
-    "send": "0.13.0",
-    "terraform": "mobify/terraform#master"
+    "send": "^0.16.2",
+    "terraform": "mobify/terraform#WEB-2067_npm-vulnerabilities",
```

Comment:
Please merge the terraform PR first and pin this to `"terraform": "mobify/terraform#master"`